File: C#/0657. Robot Return to Origin.cs
public class Solution {
    public bool JudgeCircle(string moves) {
        int l=0;
        int u=0;
        foreach(char c in moves){
            if(c=='U'){u+=1;}
            else if(c=='D'){u-=1;}
            else if(c=='L'){l+=1;}
            else if(c=='R'){l-=1;}
        }
        return l==0 && u==0;
    }
}

File: archives/leetcode2/0657. Robot Return to Origin.py
class Solution:
    def judgeCircle(self, moves: 'str') -> 'bool':
        cnt_U,cnt_L=0,0
        for s in moves:
            if s=='U':
                cnt_U+=1
            elif s=='D':
                cnt_U-=1
            elif s=='L':
                cnt_L+=1
            else:
                cnt_L-=1
        return cnt_U==0 and cnt_L==0
